qupulse/hardware/awgs/base.py

```diff
@@ -162,17 +166,26 @@ def __str__(self) -> str:
                " Use force to overwrite.".format(self.name)
 
 
+AllowedProgramTypes = Union[Loop,Sequence[LinSpaceNode],]
+
+class _ProgramType(Enum):
+    FSP = -1
```

Comment:
Waht is FSP?

Reply:
was a hacky way to introduce a "FixedStructureProgram" into the HDAWG (... a program with a fixed seqc-structure), the remaining necessary code would only be stored away in `qupulse_hdawg`